### DISCUSSION TITLE
I am unable to use the latest GitHub Copilot

### DISCUSSION THREAD

**Discussion by admin8756** [community/community#53412] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

GitHub Copilot is available on my computer with v1.78.9758. But when I updated to 1.83.41, my GitHub Copilot was unavailable.
![image](https://user-images.githubusercontent.com/21336637/233642481-40d873df-f753-40f9-97ca-acff739a20cc.png)
**This is the version of my vscode**
![image](https://user-images.githubusercontent.com/21336637/233642714-875ee460-09ff-4ec7-93c7-a527c356c5c0.png)

**Comment by phillies** [comment]
Similar problem here with 1.83.41

Runtime status (in extension view) says:
Not yet activated.
Uncaught Errors (1)
abort(CompileError: WebAssembly.instantiate(): expected magic word 00 61 73 6d, found 00 00 00 00 @+0). Build with -s ASSERTIONS=1 for more info.
Messages (1)
Command `interactiveEditor.start` appears multiple times in the `commands` section.

Downgrade to 1.82 as workaround.

**Reply by admin8756** [reply, marked as answer]
This issue was resolved in v1.84.51.
